manual/manual/library/Makefile

```diff
@@ -90,7 +95,7 @@ $(INTF): $(MLIS)
 	mv -f Ast_mapper.tex Astmapper.tex
 
 Tk.tex: tk.mli
-	$(CSLDIR)/byterun/ocamlrun $(CSLDIR)/ocamldoc/ocamldoc -latex \
+	$(OCAMLDOC) -latex \
```

Comment:
(I realize my reply above seems grumpy; I wasn't particularly, but a smiley would have helped! Thanks for the review and your general care-taking work for the manual.)